test/common.js

```diff
@@ -0,0 +1,215 @@
+import {
+	defaultLocale,
+	getDocumentLocaleSettings,
+	getLanguage,
+	supportedBaseLocales,
+	supportedLocales
+} from '../lib/common.js';
+
+var expect = chai.expect;
+
+describe('common', () => {
+
+	const htmlElem = window.document.getElementsByTagName('html')[0];
+
+	const documentLocaleSettings = getDocumentLocaleSettings();
+
+	afterEach(() => {
+		htmlElem.removeAttribute('lang');
+		htmlElem.removeAttribute('data-lang-default');
+		htmlElem.removeAttribute('data-intl-overrides');
+		htmlElem.removeAttribute('data-timezone');
+		documentLocaleSettings.reset();
+	});
+
+	it('should default to "en"', () => {
+		expect(defaultLocale).to.equal('en');
+	});
+
+	describe('getLanguage', () => {
```

Comment:
A lot of these tests are from core -- I'll remove them from there once it's updated to use this.